planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -411,14 +411,9 @@ class PlanningSceneMonitor : private boost::noncopyable
   planning_scene::PlanningSceneConstPtr scene_const_;
   planning_scene::PlanningScenePtr      parent_scene_; /// if diffs are monitored, this is the pointer to the parent scene
   boost::shared_mutex                   scene_update_mutex_; /// mutex for stored scene
-  ros::Time                             last_update_time_; /// Last time the scene was updated
-  ros::Time                             last_robot_motion_time_; /// Last time the robot has moved
-  bool                                  enforce_next_state_update_;
 
```

Comment:
The commit message makes it sound as if this makes the change ABI compatible.
Clearly, it does not. Overall, you added 4 members and renamed one.
Changing the size of a class is ABI breaking, and there's no way around it to fix this issue.
I agree with the idea of the request, but I want to stress the point that we have to send a notification on that change to moveit-users and likely also ros-users once this is released.